contracts/UFragmentsPolicy.sol

```diff
@@ -37,16 +38,27 @@ contract UFragmentsPolicy is Ownable {
         uint256 timestampSec
     );
 
+    event LogRebaseV2(
+        uint256 indexed epoch,
+        uint256 exchangeRate,
+        uint256 targetRate,
+        int256 requestedSupplyAdjustment,
+        uint256 timestampSec
+    );
+
     IUFragments public uFrags;
 
-    // Provides the current CPI, as an 18 decimal fixed point number.
+    // Provides the cpi adjusted price target, as an 18 decimal fixed point number.
     IOracle public cpiOracle;
 
     // Market oracle provides the token/USD exchange rate as an 18 decimal fixed point number.
     // (eg) An oracle value of 1.5e18 it would mean 1 Ample is trading for $1.50.
     IOracle public marketOracle;
 
-    // CPI value at the time of launch, as an 18 decimal fixed point number.
+    // @notice DEPRECATED.
```

Comment:
You could also rename the variable itself to something like `baseCpi_DEPRECATED`. This shouldn't impact storage layout, and it makes it clear if future code accidentally references it.

Reply:
I was reluctant to run an upgrade with `--force` but seems like they now allow a clean way to rename variables. 
https://github.com/OpenZeppelin/openzeppelin-upgrades/pull/531